examples/test_tesla_impact_report/test_gpt4o.ipynb

```diff
@@ -0,0 +1,373 @@
+{
```

Comment:
<blockquote>By turning on gpt4o, we use GPT-4o multimodal capabilities to do document parsing per page.</blockquote><p> </p><p>By turning on gpt4o, we use GPT-4o multimodal capabilities to do document parsing per page instead of the LlamaParse normal pipeline.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_parse/pull/180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>